README.md

```diff
@@ -0,0 +1,24 @@
+# ros2cli
+
+This repository contains the source code for ROS 2 command line interface tools included with a standard install of any ROS 2 distro. 
```

Comment:
Well except for [`sros2`](https://github.com/ros2/sros2)-related commands. Maybe there are others ??
CLI tools that are part of the standard install but not held here should be listed.

Reply:
[ros2launch](https://github.com/ros2/launch_ros/tree/master/ros2launch), [ros2test](https://github.com/ros2/ros_testing/tree/master/ros2test) too. I don't know if I'm missing another extension.

Reply:
[ros2bag](https://github.com/ros2/rosbag2/tree/master/ros2bag)